### PR TITLE
reporting everywhere

### PR DESCRIPTION
**Pull request by elimelt**
Author: Elijah

## What changes were made?

Can now report from a person's profile, a question, or interview itself

Merge with https://github.com/swecc-uw/swecc-server/pull/61

### CODE REVIEW

**Comment by elimelt** [comment]
don't review yet

**Comment by elimelt** [comment]
diff got absolutely screwed, only really need to review report related changes in components where I added the Modal

File: src/pages/ViewInterviewPage.tsx [hunk 1]
```diff
@@ -94,6 +94,9 @@ const InterviewView = ({ interview }: { interview: HydratedInterview }) => {
         <ModalContent maxH="800px" maxW="700px">
           <ModalCloseButton />
           <ReportPopUp
+            title="Report Interview"
+            badgeColorScheme="red"
+            reasonPlaceholder="Enter reason for reporting this interview"
```
Comment (ShawnCollinge) [inline src/pages/ViewInterviewPage.tsx:99]: Your reason for all of them is "Enter reason for reporting this <report type>". We can probably get rid of this and just fill in the report type part with your report type.

File: src/pages/ViewInterviewPage.tsx [hunk 2]
```diff
@@ -94,6 +94,9 @@ const InterviewView = ({ interview }: { interview: HydratedInterview }) => {
         <ModalContent maxH="800px" maxW="700px">
           <ModalCloseButton />
           <ReportPopUp
+            title="Report Interview"
+            badgeColorScheme="red"
```
Comment (ShawnCollinge) [inline src/pages/ViewInterviewPage.tsx:98]: It looks like all are red? We should just default to red and allow the option to pass in different colours if needed.

File: src/pages/ViewInterviewPage.tsx (hunk 1 above)
Comment (elimelt) [reply src/pages/ViewInterviewPage.tsx:99]: can make it default prop

File: src/pages/ViewInterviewPage.tsx (hunk 2 above)
Comment (elimelt) [reply src/pages/ViewInterviewPage.tsx:98]: yes can add default

Comment (elimelt) [reply src/pages/ViewInterviewPage.tsx:98]: oh alr set as default, will remove

**Review by ShawnCollinge — APPROVED** [review]
LGTM

**Pull request merged**